### PR TITLE
Drink flavors

### PR DESCRIPTION
**Pull request by Aikakakah**
## About the PR
Added drink flavors. No more "tastes like alcohol."

## Why / Balance
Why? For spite. No longer shall we be fooled about the contents of a bottle when it all tastes the same. We needed variety and so I give you _flavor._

## Technical details
Added the flavor profiles into the Delta-V namespace, and commented the changes I had to make on the upstream drink files.

## Requirements

- [X] I have tested all added content and changes.
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

- tweak: Added some flavor to the drinks.

### CODE REVIEW

File: Resources/Prototypes/DeltaV/Reagents/Consumable/Drink/drinks.yml [hunk 1]
```diff
@@ -171,7 +171,7 @@
   parent: BaseDrink
   desc: reagent-desc-kvass
   physicalDesc: reagent-physical-desc-bubbly
-  flavor: bread
+  flavor: kvass                     #Delta-V Flavor additions
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Reagents/Consumable/Drink/drinks.yml]: holy spaces batman
for all these just 1 space before the #

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Review by deltanedas — DISMISSED** [review]

**Comment by deltanedas** [comment]
remove xeno basher as its failing the linter, also awaiting direction input

**Comment by Lyndomen** [comment]
ETA 24 hours on Direction review

**Review by Lyndomen — DISMISSED** [review]
Approved broadly, there are some ruminations about descriptions being a bit too indescript, but frankly descriptions everywhere are a mess. So this is good for now, and will likely get iterated upon.

**Comment by deltanedas** [comment]
tests still fail

**Review by deltanedas — APPROVED** [review]

**Pull request merged**